contributing/code/bugs.rst

```diff
@@ -14,8 +14,9 @@ Before submitting a bug:
 * Double-check the official :doc:`documentation </index>` to see if you're not misusing the
   framework;
 
-* Ask for assistance on `Stack Overflow`_ or on the #symfony `IRC channel`_
-  if you're not sure if your issue is really a bug.
+* Ask for assistance on `Stack Overflow`_ on the #symfony `IRC channel`_, or on
+  the #support channel of `the Symfony Slack` if you're not sure if your issue
```

Comment:
Maybe we could add the link of the[ symfony slack](https://symfony.com/slack-invite)